packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -36,6 +36,34 @@ BuySellModalFooter.propTypes = {
     onSubmit: PropTypes.func.isRequired,
 };
 
+const generateModalTitle = (formik_ref, my_profile_store, table_type, selected_ad) => {
```

Comment:
Declared the function outside the `BuySellModal` to avoid it being mounted and re-mounted:
https://levelup.gitconnected.com/code-review-avoid-declaring-react-component-inside-parent-component-1768a645f523